### ISSUE TITLE
stryker-javascript-mutator fails to load in node 4

### ISSUE THREAD

**Issue by jmpresso** [stryker-mutator/stryker#593]
### Summary

Locally, stryker-javascript-mutator works for me with Node 8. With Node 4, it fails to load.

Running "stryker:sauceLabsRun" (stryker) task
[DEBUG] ConfigReader - [39mLoading config stryker.conf.js
[WARN] PluginLoader - [39mError during loading "stryker-javascript-mutator" plugin:
Cannot find module 'lodash'

### Stryker config

I have some additional grunt config, but I don't think it's relevant.

```js
config.set({
        testRunner: 'karma',
        testFramework: 'jasmine',
        reporter: ['html', 'clear-text', 'event-recorder'],
        mutator: 'javascript',
        files: [
            {
                pattern: 'src/**/*.js',
                included: true,
                mutated: true
            },
            'test/**/!(*e2e)spec.js',
            {
                pattern: 'test/**/*.json',
                included: false,
                mutated: false
            }
        ],
        logLevel: 'info',
        thresholds: { high: 99, low: 94, break: 90 }
    });
}
```

### Stryker environment

```
    "grunt-sonar-runner": "2.4.3",
    "stryker": "^0.17.0",
    "stryker-api": "^0.12.0",
    "stryker-html-reporter": "^0.11.4",
    "stryker-jasmine": "^0.7.3",
    "stryker-javascript-mutator": "^0.3.1",
    "stryker-karma-runner": "^0.11.3",
```

### Your Environment
| software         | version(s)
| ---------------- | -------  
| node             |  4.x
| npm              |  2.x
| Operating System |  RHEL or Windows

**Comment by jmpresso** [comment]
One thing I forgot- locally I am using a newer node version in Windows. I will try to run with the same version I'm running in Linux to make sure that's not the issue.

**Comment by nicojs** [comment]
@jmpresso thanks for reporting this issue. This is a pretty big oversight on our part, as we forgot to list one of our dependencies. Unfortunately these kind of things fall through the cracks even in our integration tests 😞 

I just did a PR that fixes this issue, we should be able to release a fix soon.

**Comment by jmpresso** [comment]
Thanks for the quick fix @nicojs !

**Comment by simondel** [comment]
This has been released in stryker-javascript-mutator@0.3.3 :) Thanks for reporting it!
